backend/src/main/java/gov/cdc/usds/simplereport/db/repository/DataHubUploadRespository.java

```diff
@@ -7,7 +7,9 @@
 import java.util.List;
 import java.util.UUID;
 
-public interface DataHubUploadRespository extends Repository<DataHubUpload, UUID> {
+public interface DataHubUploadRespository extends Repository<DataHubUpload, UUID>, AdvisoryLockManager {
+
+    int SCHEDULED_UPLOAD_LOCK = 66037627; // arbitrary 32-bit integer for our lock
```

Comment:
Same complaint from IDE linter about constant in an interface. (Even though I think it makes the most sense here.)

Reply:
I should also put a javadoc on those constants. Is there a way to tell IntelliJ to relax?

Reply:
Yup. Personally, I don't see the problem with putting constants in interfaces. Here's what the linter says:
"Reports on any constants (i.e. public static final fields) declared in interfaces. Some coding standards require that constants be declared in abstract classes instead. "